src/content/en/updates/2017/01/css-grid.md

```diff
@@ -0,0 +1,225 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: CSS Grid is a new layouting system for the web.
+
+
+{# wf_published_on: 2017-01-27 #}
+{# wf_updated_on: 2017-01-27 #}
+{# wf_featured_image: /web/updates/images/2017/01/css-grid/grid.png #}
+{# wf_tags: css,layout,grid #}
+{# wf_featured_snippet: CSS Grid is a new layouting system for the web. #}
+
+# CSS Grid – Table layout is back. Be there and be square. {: .page-title }
+
+{% include "web/_shared/contributors/surma.html" %}
+
+> *TL;DR:* If you are familiar with Flexbox, Grid should feel familiar.
+[Rachel Andrew](https://twitter.com/Rachelandrew) maintains a great
+[website dedicated to CSS Grid](http://gridbyexample.com/) to help you get
+started. Grid is now available in Google Chrome!
+
+## Flexbox? Grid?
+
+Over the past few years, CSS Flexbox has become widely used and
+[browser support](http://caniuse.com/#feat=flexbox) is looking really good
+(unless you are one of the poor souls that have to support IE9 and below).
+Flexbox made a lot of complex layout tasks easier, like equi-distant spacing
+between elements, top-to-bottom layouts or the holy grail of CSS wizardry:
+vertical centering.
+
+<img width="50%" src="/web/updates/images/2017/01/css-grid/alignproblem.png" alt="There's no way to align elements across multiple flexbox containers.">
```

Comment:
(In my defense, I copied that form one of Lewis’ articles 😉 )

I don’t really want a float, but the picture is really big otherwise. Should I include the padding _in_ the picture itself?